src/pages/eventing/EventingLogs.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from 'react';
+import {useParams} from "react-router-dom";
+import {Button, Icon, Table} from "antd";
+import '../../index.css';
+import client from "../../client";
+import Sidenav from '../../components/sidenav/Sidenav';
+import Topbar from '../../components/topbar/Topbar';
+import EventTabs from "../../components/eventing/event-tabs/EventTabs";
+import FilterForm from "../../components/eventing/FilterForm";
+import './event.css';
+//redux
+import {useDispatch, useSelector} from "react-redux";
+import {set, increment, decrement} from "automate-redux";
+import InfiniteScroll from 'react-infinite-scroller';
+
+const getIconByStatus = (status) => {
+  switch(status){
+    case "processed":
+      return <Icon type="check" style={{color: "#00FF00"}}/>
+    case "failed":
+      return <Icon type="close" style={{color: "red"}}/>
+    default:
+      return <Icon type="hourglass" />
+  }
+}
+
+function isJson(str) {
+  try {
+      JSON.parse(str);
+  } catch (e) {
+    return str;
+  }
+  const jsonObj = JSON.parse(str)
+  return jsonObj;
+}
+
+const columns = [
+  { title: 'Event ID', dataIndex: '_id', key: '_id' },
+  { title: 'Trigger name', dataIndex: 'rule_name', key: 'rule_name' },
+  { title: 'Invocations', key: 'invocations', render: record => <p>{record.invocation_logs.length}</p> },
+  { title: 'Status', key: 'status', render: record => getIconByStatus(record.status) },
+  { title: 'Created at', key: 'date', render:(record) => <p>{new Date(record.event_timestamp).toISOString()}</p> },
+];
+
+const EventingLogs = () => {
+  // Router params
+  const { projectID } = useParams()
+  const [modalVisible, setModalVisible] = useState(false);
+  const dispatch = useDispatch();
+  const eventLogs = useSelector(state => state.eventLogs);
+  const eventFilters = useSelector(state => state.uiState.eventFilters);
+  const [moreEventLogs, hasMoreEventLogs] = useState(true);
+
+  useEffect(() => {
+    dispatch(increment("pendingRequests"));
+    client.eventing.fetchEventLogs(projectID, eventFilters, 0)
+    .then(res => dispatch(set("eventLogs", res)))
+    .catch(ex => console.log(ex))
+    .finally(() => dispatch(decrement("pendingRequests")))
+  }, [eventFilters])
+  
+  const expandedInvocations = record => {
+    const requestJSON = isJson(record.request_payload);
+    const responseJSON = isJson(record.response_body); 
+    return (
+      <div>
+        <b>Request payload</b><br/>
+        <pre>{JSON.stringify(requestJSON, null, 2)}</pre><br/><br/>
+        <b>Status Code</b><br/>
+        {record.response_status_code}<br/><br/>
+        <b>Response</b><br/>
+        <pre>{JSON.stringify(responseJSON, null, 2)}</pre>
+      </div>
```

Comment:
There will be one more thing to show. `record.error_msg`. It is also a json string as request payload. And it needs to be shown only if `record.error_msg` has some value